### DISCUSSION TITLE
Geonames in Metadata

### DISCUSSION THREAD

**Discussion by mobab-th** [sepinf-inc/IPED#2186] [Ideas]
Hi,

since March 2024, exiftools has had a geolocation function that can identify the nearest city, state, county and country based on GPS position. Can't this feature be used to add geolocation names to the metadata? 

more information here: [https://exiftool.org/geolocation.html](https://exiftool.org/geolocation.html)

**Comment by wladimirleite** [comment]
I think it is a different approach, but it seems related to #1487.

**Reply by lfcnassif** [reply]
And there is a PR implementing it and waiting for review: https://github.com/sepinf-inc/IPED/pull/1890

If anyone has available time and could help reviewing and testing it, that would be very welcome.
